### ISSUE TITLE
[BUG] Spark python task does not make use of --parameters on dbx execute

### ISSUE THREAD

**Issue by kiranvasudev** [databrickslabs/dbx#48]
## Expected Behavior

Using `dbx execute --job=cicd-sample-project-sample --cluster-name=test-cluster` should take the parameters for the spark python task which is part of the `deployment.json`.

## Current Behavior
Using `dbx execute --job=cicd-sample-project-sample --cluster-name=test-cluster` **does not** take the `spark-python-task` parameters from `deployment.json`

## Steps to Reproduce (for bugs)
1. use the default deployment.json
```json
{
    "default": {
        "jobs": [
            {
                "name": "cicd-sample-project-sample",
                "new_cluster": {
                    "spark_version": "7.3.x-cpu-ml-scala2.12",
                    "node_type_id": "i3.xlarge",
                    "aws_attributes": {
                        "first_on_demand": 0,
                        "availability": "SPOT"
                    },
                    "num_workers": 2
                },
                "libraries": [],
                "email_notifications": {
                    "on_start": [],
                    "on_success": [],
                    "on_failure": []
                },
                "max_retries": 0,
                "spark_python_task": {
                    "python_file": "cicd_sample_project/jobs/sample/entrypoint.py",
                    "parameters": [
                        "--conf-file",
                        "conf/test/sample.json"
                    ]
                }
            }
       ]
}
```
2. execute the job using `dbx execute --job=cicd-sample-project-sample --cluster-name=test-cluster` on an interactive(all purpose) cluster.

## Context
I am trying to run a python project that makes use of command line arguments.

## Your Environment

* dbx version used: 0.1.3
* Databricks Runtime version: 7.3LTS

**Comment by renardeinside** [comment]
hi @kiranvasudev , 
I'll take care of this issue, thanks a lot for raising it.

**Comment by renardeinside** [comment]
hi @kiranvasudev , thanks a lot for raising the issue - closed in #56 , will be released in 0.1.4

**Comment by kiranvasudev** [comment]
@renardeinside Thanks a lot !
